.validate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "scripts": {
+    "lint": "jshint ."
+  },
+  "pre-commit": ["lint", "validate", "test"]
+}
```

Comment:
do you prefer the `.validate.json` over just slapping this content in `package.json`?  i've always leaned towards the latter, but no hard bias